packages/terra-demographics-banner/src/_LargeDemographicsBannerDisplay.jsx

```diff
@@ -42,18 +46,46 @@ export default (props) => {
     ),
     customProps.className,
   );
+  
+  if (customProps.className) {
+    mainClasses = `${mainClasses} ${customProps.className}`;
+  }
```

Comment:
Here's an example of a cleaner pattern to combine those classes that we've been following as part of our theme context uplift
https://github.com/cerner/terra-core/blob/master/packages/terra-card/src/Card.jsx#L45-L52

Reply:
This has been updated.